strawberry/flask/tests/test_graphqlview.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+import pytest
+
+from flask import url_for
+
+from .app import init_app
+
+
+@pytest.fixture
+def app():
+    return init_app()
```

Comment:
can we move the contents of init_app here? feels redundant to have another file for this ;)

we could move this fixture tuore flast/tests/conftest.py as well :)

Reply:
Sure. I will move it to `init_app` method